### PR TITLE
move sdk section

### PR DESCRIPTION
**Pull request by joan-ing**
Closes #

## Summary

-

---

### Remaining work

The following work still needs to be completed:

- [ ] <List outstanding tasks here>

### Dependencies

- [ ] <Tasks that must be completed before merging this pull request>

---

### Timing

**Release**:

- [x] When ready
- [ ] Hold for release | Release after: <mm/dd/yy>

### CODE REVIEW

**Comment by philkra** [comment]
@joan-ing  I added all Python examples

File: 040-Developer-guides/020-Python-SDK/020-examples.mdx [hunk 1]
```diff
@@ -184,49 +210,56 @@ records = resp.json()
...
+
 ```python
 from xata.client import XataClient
 records = XataClient().records()
 
 record = records.deleteRecord("Avengers", "captain-america")
 print(record.status_code)
 # 204
-```
+````
```
Comment (snide) [inline 040-Developer-guides/020-Python-SDK/020-examples.mdx:238]: This shouldn't be here and is likely copy/paste from a legacy docs source.

File: 040-Developer-guides/020-Python-SDK/020-examples.mdx [hunk 2]
```diff
@@ -184,49 +210,56 @@ records = resp.json()
 print(data["records"])
 # [{"id": "spidey", "name": "Peter Parker", "job": "spiderman"}]
 # Note it will be an array, even if there is only one record matching the filters
-```
+````
```
Comment (snide) [inline 040-Developer-guides/020-Python-SDK/020-examples.mdx:213]: This shouldn't be here and is likely copy/paste from a legacy docs source.

File: 040-Developer-guides/020-Python-SDK/020-examples.mdx [hunk 3]
```diff
@@ -148,27 +169,32 @@ print(spiderman.json())
...
 })
 assert resp.is_success()
 print(resp["records"])
+
 # [{"id": "spidey", "name": "Peter Parker", "job": "spiderman"}]
+
 # Note it will be an array, even if there is only one record matching the filters
-```
+
+````
```
Comment (snide) [inline 040-Developer-guides/020-Python-SDK/020-examples.mdx:196]: This shouldn't be here and is likely copy/paste from a legacy docs source.

File: 040-Developer-guides/020-Python-SDK/020-examples.mdx [hunk 4]
```diff
@@ -148,27 +169,32 @@ print(spiderman.json())
 batman = records.getRecord("Avengers", "bruce-wayne")
 print(batman.status_code)
 # 404
-```
+````
```
Comment (snide) [inline 040-Developer-guides/020-Python-SDK/020-examples.mdx:172]: This shouldn't be here and is likely copy/paste from a legacy docs source.

File: 040-Developer-guides/020-Python-SDK/020-examples.mdx [hunk 5]
```diff
@@ -111,31 +122,41 @@ record["job"] = "the incredible hulk"
...
 print(batman.status_code)
+
 # 404
+
 print(batman.is_success())
+
 # False
-```
+
+````
```
Comment (snide) [inline 040-Developer-guides/020-Python-SDK/020-examples.mdx:158]: This shouldn't be here and is likely copy/paste from a legacy docs source.

File: 040-Developer-guides/020-Python-SDK/020-examples.mdx [hunk 6]
```diff
@@ -111,31 +122,41 @@ record["job"] = "the incredible hulk"
 resp = client.records().upsertRecordWithID("Avengers", "spidey-1", record)
 # On update status code = 200
 assert resp.status_code == 200
-```
+````
```
Comment (snide) [inline 040-Developer-guides/020-Python-SDK/020-examples.mdx:125]: This shouldn't be here and is likely copy/paste from a legacy docs source.

File: 040-Developer-guides/020-Python-SDK/020-examples.mdx [hunk 7]
```diff
@@ -34,46 +35,56 @@ Below are the parameters passed to the methods in the following examples:
...
 record["job"] = "the incredible hulk"
 resp = xata.records().upsert_with_id("Avengers", "spidey-1", record)
+
 # On update status code = 200
+
 assert resp.status_code == 200
 assert resp.is_success()
-```
+
+````
```
Comment (snide) [inline 040-Developer-guides/020-Python-SDK/020-examples.mdx:86]: This shouldn't be here and is likely copy/paste from a legacy docs source.

File: 050-Xata-Actions/040-update.mdx [hunk 8]
```diff
@@ -210,6 +269,18 @@ const updatedUser = await xata.db.Users.update(
 );
 ```
 
+```python
+user = xata.records().update(
+  "Users", 
+  "rec_c8hnbch26un1nl0rthkg", 
+  {
+    "name": 'Keanu Reaves',
```
Comment (snide) [inline 050-Xata-Actions/040-update.mdx:277]: Reeves

File: 050-Xata-Actions/040-update.mdx [hunk 9]
```diff
@@ -55,12 +67,18 @@ user.update({ email: 'newemail@example.com' });
 
 It's also possible to update a record if it exists, or create it if it doesn't exist:
 
-<TabbedCode tabs={['TypeScript', 'JSON']}>
+<TabbedCode tabs={['TypeScript', 'Python', 'JSON']}>
 
 ```ts
 const user = await xata.db.Users.createOrUpdate('myid', { name: 'Keanu Reaves' });
```
Comment (snide) [inline 050-Xata-Actions/040-update.mdx:73]: Reeves

File: 050-Xata-Actions/040-update.mdx [hunk 10]
```diff
@@ -195,11 +254,11 @@ The `version` field can be used to perform optimistic concurrency control, also
...
 
-<TabbedCode tabs={['TypeScript', 'JSON']}>
+<TabbedCode tabs={['TypeScript', 'Python', 'JSON']}>
 
 ```ts
 const updatedUser = await xata.db.Users.update(
-  'myid1',
+  'rec_c8hnbch26un1nl0rthkg',
   {
     name: 'Keanu Reaves',
```
Comment (snide) [inline 050-Xata-Actions/040-update.mdx:263]: Reeves

**Comment by philkra** [comment]
Thank you Dave for review and feedback. I'll tackle these 3 items:

- [ ] The JSON examples all have different formats. Sometimes we use variables {db}, sometimes we use holders /tutorial/. I'd suggest using variables everywhere because it makes it clear what would need to be changed and matches the REST API docs.
- [x] There are several places where four backticks are used, rather than three. My guess is that this content was copied from an older version of the docs before <TabbedCode/> was in place.
- [x] Might want to run a search and replace for Keanu if we're using that example. We're back and forth with his spelling throughout the docs.

File: 040-Developer-guides/001-overview.mdx [hunk 11]
```diff
@@ -0,0 +1,10 @@
+---
+title: SDK overview
+navTitle: Overview
+keywords: ['python', 'sdk', 'typescript', 'json']
+description: Get to know Xata using SDKs
+slug: sdk/overview
+published: true
+---
+
+Xata supports the following SDKs
```
[the remark this answers is not in the document]
Comment (philkra) [reply 040-Developer-guides/001-overview.mdx]: page removed

**Review by snide — APPROVED** [review]

**Review by philkra — APPROVED** [review]
LGTM

**Pull request merged**